java/release_notes.txt

```diff
@@ -1,3 +1,22 @@
+Sep 07, 2016: libphonenumber-7.6.1
+Code changes:
+ - Phonemetadata.java has two more fields to represent possible lengths of phone
+   numbers. Changed BuildMetadataFromXml (and the resultant jar) to alter the way
+   that metadata about possible-lengths information is consumed when constructing
+   metadata to populate these.
+   Discussion list email:
+   https://groups.google.com/forum/#!topic/libphonenumber-discuss/75TOpTFVi08
+Metadata changes:
```

Comment:
Squashed and merged.

2016-09-07 16:54 GMT+02:00 penmetsaa <notifications@github.com>:

> In java/release_notes.txt
> <https://github.com/googlei18n/libphonenumber/pull/1313#discussion_r77838395>
> :
>
> > @@ -1,3 +1,22 @@
> > +Sep 07, 2016: libphonenumber-7.6.1
> > +Code changes:
> > + - Phonemetadata.java has two more fields to represent possible lengths of phone
> > +   numbers. Changed BuildMetadataFromXml (and the resultant jar) to alter the way
> > +   that metadata about possible-lengths information is consumed when constructing
> > +   metadata to populate these.
> > +   Discussion list email:
> > +   https://groups.google.com/forum/#!topic/libphonenumber-discuss/75TOpTFVi08
> > +Metadata changes:
>
> Done Lara. PTAL
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/googlei18n/libphonenumber/pull/1313/files/d3cd5fe7f7646d7a3308581e93a2b2b50da747fa#r77838395>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIjis7V3D3RJFSDTYf_uoc0O94PiV870ks5qntAUgaJpZM4J2zHp>
> .
>